docs/platform/what-is-tizen/versions/tizen-6-5-m2.md

```diff
@@ -0,0 +1,142 @@
+# Tizen 6.5
+
+The second milestone (M2) release of Tizen 6.5 was announced in October 2021.
+Tizen is an open-source operating system (OS) maintained by Samsung and is developed and optimized for smart devices since 2012.
+
+Tizen 6.5 achieved to connect more devices and create intelligent services on those connected devices.
+New multi-device experience frameworks are introduced in Tizen 6.5.
+On top of that, Tizen app development becomes easier and more familiar by introducing new popular cross-platform framework, flutter for Tizen.
+We are also enhancing previously introduced .NET and web with new updates.
+We made Tizen 6.5 more flexible and adaptable to IoT devices.
+With enhanced Tizen Image Creator, you can create custom Tizen image with just a few clicks.
+
+## Multiple device experiences
+
+- **Advanced companion framework**:
+
+The Advanced Companion Framework facilitates multi-device application development,
```

Comment:
Recommend to align the text bellow with the title **Advanced companion framework**: